modules/parquet/test/data/files.js

```diff
@@ -6,23 +6,23 @@ const PARQUET_FILES = [
   {supported: true, title: 'alltypes_plain', path: 'good/alltypes_plain.parquet'},
   {supported: true, title: 'alltypes_plain_snappy', path: 'good/alltypes_plain.snappy.parquet'},
   {supported: true, title: 'binary', path: 'good/binary.parquet'},
-  {supported: false, title: 'bloom_filter', path: 'good/bloom_filter.bin'},
+  {supported: false, title: 'bloom_filter', path: 'good/bloom_filter.bin'}, // It's not a parquet file should we remove it?
```

Comment:
Hmm... This looks like a specialized Dict for very large dictionaries: https://github.com/apache/parquet-format/blob/master/BloomFilter.md

For now, I think we can comment out this line with that link in the comment.

Reply:
Done